content/en/mobile/push_notification.md

```diff
@@ -0,0 +1,131 @@
+  tag: "Documentation"
+  text: "Incident Notification Rules Documentation"
+- link: "https://docs.datadoghq.com/getting_started/workflow_automation/"
+  tag: "Documentation"
+  text: "Workflow Automation Documentation"
+---
+
+Receive mobile push notifications for [on-call alerts](#circumvent-mute-and-Do-Not-Disturb-mode-for-On-Call), [incidents](#incident-notifications), and [workflow automation updates](#workflow-automation-notifications), so you can stay informed in real time from the Datadog mobile app.
+
+## Set up push notifications
+<div class="alert alert-info">
+When you log into the Datadog mobile app for the first time, an onboarding flow takes care of notification settings and permissions.
+</div>
+
+However, by default, the mobile app is not allowed to send you notifications. To receive push notifications: 
+
+{{< tabs >}}
+{{% tab "iOS" %}}
+
+1. In the Datadog mobile app, navigate to **Settings** > **Notifications**.
+
+   {{< img src="service_management/mobile/ios_settings_may_2025.png" alt="Find the notification settings in the iOS version of Datadog's mobile app." style="width:35%;" >}}
+
+2. Enable the **Allow Notifications** toggle. If this is your first time enabling notifications, this opens up a permissions prompt. Grant permission, then touch **Enable Notifications** again to go to the iOS system settings.
+
+   {{< img src="service_management/mobile/ios_notification_may_2025.png" alt="Configure the system notification settings of your iOS device." style="width:100%;" >}}
+
+3. Within the iOS system settings, make sure you enable the **Allow Notifications** toggle. Datadog highly recommends you also enable the **Sound** and **Badges** toggles.
+
+Make sure you grant the mobile app the necessary permissions.
+{{% /tab %}}
+
+{{% tab "Android" %}}
+1. In the Datadog mobile app, navigate to **Settings** > **Notifications**.
+
+   {{< img src="service_management/mobile/android_settings_may_2025.png" alt="Find the notification settings in the Android version of Datadog's mobile app." style="width:35%;" >}}
+
+2. Enable the **Allow notifications** toggle. Datadog highly recommends you also enable **Sound and vibration** and **Show content on Lock screen**.
+
+   {{< img src="service_management/mobile/android_notification_may_2025.png" alt="Configure the system notification settings of your Android device." style="width:100%;" >}}
+
+{{% /tab %}}
+{{< /tabs >}}
+
+### Custom sounds
+On both iOS and Android, you have the option to override the default system notification sounds. The Datadog app comes preloaded with a selection of custom sounds.  
+
+## Circumvent mute and Do Not Disturb mode for On-Call
+You can override your device's system volume and Do Not Disturb mode for both push notifications (from the Datadog mobile app) and telephony notifications (such as voice call and SMS).
+
+For more information, here is a full guide on [how to set up your mobile device for On-Call][]
```

Comment:
```suggestion
For more information, see the [guide on setting up your mobile device for On-Call][].
```
This link seems to be missing, did you mean to link to `/service_management/on-call/guides/configure-mobile-device-for-on-call`?